### ISSUE TITLE
select max, first, last does not return timestamp

### ISSUE THREAD

**Issue by rishid** [influxdb/influxdb#1549]
Influxdb v0.8.8 - max, last and first aggregate functions do not return the time associated with the point

e.g.
```
select max(value) from /servers.web01.host.diskspace.root.usage/ where time > now() - 7d 
servers.web01.host.diskspace.root.usage
                time                 max
            0.000000            1.092457
```

**Comment by jeremyVignelles** [comment]
Hi,
Is this issue fixed? I have the same issue on v0.9.1 using LAST()

**Comment by beckettsean** [comment]
#1577 is the 0.9.x version of the issue. Please follow that for status.
